### ISSUE TITLE
PTAProblemParser Error

### ISSUE THREAD

**Issue by Just-Prog** [jmerle/competitive-companion#295]
Example: [6-1 简单输出整数](https://pintia.cn/problem-sets/14/exam/problems/733)

Console log:

```        
TypeError: Cannot read properties of null (reading 'querySelector')
    at PTAProblemParser.parse (common.js:1:391495)
    at l (content.js:1:320)
    at content.js:1:1263
```

**Comment by Just-Prog** [comment]
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/62735122/197687992-9e6c5527-b7cf-4d2d-8882-42947cf8267d.png">

**Comment by jmerle** [comment]
Fixed in fbed80530368b20381bab12873d7467c2ae19198, will be released within a few hours.

PTA may be updating their UI, but the HTML elements the parser is interested in are still mostly named and structured the same, so I don't think we have to wait for the UI to stabilize. Feel free to open a new issue when a future PTA UI update breaks the parser again.

**Comment by Just-Prog** [comment]
> Fixed in [fbed805](https://github.com/jmerle/competitive-companion/commit/fbed80530368b20381bab12873d7467c2ae19198), will be released within a few hours.
> 
> PTA may be updating their UI, but the HTML elements the parser is interested in are still mostly named and structured the same, so I don't think we have to wait for the UI to stabilize. Feel free to open a new issue when a future PTA UI update breaks the parser again.

Thanks for your hard work ❤️
